lib/dash/dash_parser.js

```diff
@@ -222,16 +221,13 @@ shaka.dash.DashParser = class {
    */
   async requestManifest_() {
     const requestType = shaka.net.NetworkingEngine.RequestType.MANIFEST;
-    const advType = shaka.net.NetworkingEngine.AdvancedRequestType.MPD;
+    const type = shaka.net.NetworkingEngine.AdvancedRequestType.MPD;
     const request = shaka.net.NetworkingEngine.makeRequest(
         this.manifestUris_, this.config_.retryParameters);
     const networkingEngine = this.playerInterface_.networkingEngine;
-
-    const format = shaka.util.CmcdManager.StreamingFormat.DASH;
-    this.playerInterface_.modifyManifestRequest(request, {format: format});
-
     const startTime = Date.now();
-    const operation = networkingEngine.request(requestType, request, advType);
+    const operation = networkingEngine.request(
+        requestType, request, {type: type});
```

Comment:
You can just do `{type}`, since the variable names are the same. That would also make this short enough to fit in one line again.

Reply:
I didn't see object property shorthand being used anywhere else. Is this feature available to us in closure compiler?

Reply:
Seems like it does.